modules/Users/language/en_us.lang.php

```diff
@@ -384,8 +384,7 @@
     'LBL_WIZARD_FINISH_BUTTON' => 'Finish',
     'LBL_WIZARD_FINISH_TITLE' => 'You are ready to use SuiteCRM!',
 
-    'LBL_WIZARD_FINISH' => 'Click <b>Finish</b>
-below to save your settings and to begin using SuiteCRM. For more information on using SuiteCRM:<br /><br />
+    'LBL_WIZARD_FINISH' => 'Click <b>Finish</b>below to save your settings and to begin using SuiteCRM. For more information on using SuiteCRM:<br /><br />
```

Comment:
there needs to be a space between the "\</b\>" and "below". this prints out Finishbelow

Reply:
done!